### PR TITLE
Track List story

### PR DESCRIPTION
**Pull request by sarru1291**
**Description**

> What does this PR achieve?
User interface story for the only tracklist.
Screenshot from 2019-04-25 09-22-16

### CODE REVIEW

**Comment by Batey96** [comment]
Great but i think what @isa wants is for you to maintain the data flow from the current code strucuture..
There is already a tracklist component..the goal here is to have a choice to use the youtube component over the current embedded one..All things being equal

**Comment by sarru1291** [comment]
> Great but i think what @isa wants is for you to maintain the data flow from the current code strucuture..
> There is already a tracklist component..the goal here is to have a choice to use the youtube component over the current embedded one..All things being equal

I was testing the tracklist that's why I made its story. Currently, I'm about to complete the youtube component.

File: packages/ia-components/sandbox/theatres/components/track-list/track-list.stories.js [hunk 1]
```diff
@@ -1 +1,40 @@
 import style from './track-list.less';
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, number } from '@storybook/addon-knobs';
+import TrackList from './track-list';
+
+ storiesOf('Theatres', module)
+  // .addParameters({ jest: ['example.test.js'] })
```
Comment (iisa) [inline packages/ia-components/sandbox/theatres/components/track-list/track-list.stories.js]: let's remove this

**Comment by sarru1291** [comment]
it's resolved now.

**Pull request merged**